src/components/ChatWindow/ChatWindow.tsx

```diff
@@ -39,17 +40,26 @@ const ChatWindow = ({ onChatActivation }: Props) => {
 
     // Send the message to the server
     try {
+      setIsLoading(true)
       const response = await ChatAPI.send({
         botId: BOT_ID,
         chatSessionId: CHAT_SESSION_ID,
         command: message,
       })
 
       // Handle the response from the server and update chat history
-      console.log("response", response)
       addMessageToChatHistory(response.output, "bot")
-    } catch (error) {
+    } catch (error: unknown) {
```

Comment:
```suggestion
    } catch (error: any) {
```
```suggestion
    } catch (error: Dict) {
```
```suggestion
    } catch (error: Error) { // this Error type is coming from React & Typescript
```

try one of these instead of using `unknown`

Reply:
Anh có thử dùng `Dict` và `Error` nhưng eslint báo nên dùng `any` hoặc `unknown`
<img width="689" alt="image" src="https://github.com/AI-ChatBot-Yoga/ai-chatbot/assets/133303660/99cbd0e4-bc0f-4c6d-877d-aa6242a32240">

khi dùng `any` thì eslint nó warning lỗi sau 
<img width="701" alt="image" src="https://github.com/AI-ChatBot-Yoga/ai-chatbot/assets/133303660/42c63099-0147-4df9-9b5f-ed3be9f0dcd7">

Reply:
Should I change to `any` or stick with `unknown`?

Reply:
@vuson1709 it's ok in this case, we can comeback to it later if there's an error